vsphere_plugin_common/__init__.py

```diff
@@ -1664,14 +1718,23 @@ def create_server(
         relospec.datastore = datastore.obj
         relospec.pool = resource_pool.obj
         if not auto_placement:
-            logger().warn(
+            self._logger.warn(
                 'Disabled autoplacement is not recomended for a cluster.'
             )
             relospec.host = host.obj
 
+        # get list nic mac's for remove
+        macs_for_remove = []
+        if postpone_delete_networks and not enable_start_vm:
```

Comment:
So if postpone_delete_networks is false, that sounds like the networks should still be deleted but earlier than they otherwise would be. Is this the case?

Reply:
Yes, if postpone_delete_networks is false, we will remove network interfaces as usual.